packages/frontend-v2/components/Sidebar/Sidebar.tsx

```diff
@@ -18,7 +18,11 @@ import { handleApiClientError } from "../../utils";
 import axios from "axios";
 import { useRouter } from "next/router";
 import { useMajor } from "../../hooks/useMajor";
-import { WorkerMessage, WorkerMessageType, WorkerPostInfo } from "../../validation-worker/worker-messages";
```

Comment:
this file has a lotta prettier changes. only actual change is adding the COOP_BLOCK constant course so that we don't have magic strings in the codebase